lib/gridium/version.rb

```diff
@@ -1,3 +1,3 @@
 module Gridium
-  VERSION = "1.0.1"
+  VERSION = "1.0.#{ENV['BUILD_NUMBER']}"
```

Comment:
This fails instead of creating gridium-1.0.#{ENV['BUILD_NUMBER']} when doing gem build locally. Nice @_@